clients/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/spring/client/properties/common/ZeebeClientProperties.java

```diff
@@ -22,13 +22,14 @@
 import java.util.Map;
 import org.springframework.boot.context.properties.DeprecatedConfigurationProperty;
 import org.springframework.boot.context.properties.NestedConfigurationProperty;
+import org.springframework.util.unit.DataSize;
 
 @Deprecated(forRemoval = true, since = "8.8")
 public class ZeebeClientProperties extends ApiProperties {
   private Integer executionThreads;
   private Duration messageTimeToLive;
-  private Integer maxMessageSize;
-  private Integer maxMetadataSize;
+  private DataSize maxMessageSize;
+  private DataSize maxMetadataSize;
```

Comment:
❓ I haven't checked in detail but I understand this is not breaking as DataSize supports parsing just bare numbers, thus previous bare byte numbers just work still, right?

Reply:
Yes, this setting is non-breaking as plain numbers will be interpreted as bytes which is exactly what we did before.